requirements.txt

```diff
@@ -273,3 +273,13 @@ vine==1.1.4 \
 freezegun==0.3.9 \
     --hash=sha256:8842688de9497c82ecb40c30274ecc676e97ac78765b0ade105a97063d5b7a11 \
     --hash=sha256:783ccccd7f60968bfe49ad9e114c18ea2b63831faaaf61c1f1f71ddfde1c0eee
+cookies==2.2.1 \
+    --hash=sha256:15bee753002dff684987b8df8c235288eb8d45f8191ae056254812dfd42c81d3
+responses==0.7.0 \
+    --hash=sha256:860f2ac3aed6954b6dfc79ee27a229d423c77ad7a4c242f26c1489bdd702d8e4
+pytest==3.2.1 \
```

Comment:
Oops, I must have missed that. I added these when I was trying to get the cache clearing working. I guess we don't need them, will take this out.